build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -122,7 +122,8 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                         "debian.x64.version",
                         "centos.x64.version",
                         "fedora.23.x64.version",
-                        "opensuse.13.2.x64.version"
+                        "opensuse.13.2.x64.version",
+                        "opensuse.42.1.x64.version"
```

Comment:
Yeah, it's going to be in a separate PR. We also need a new Dockerfile and some changes to the debian packaging scripts.